GNUmakefile

```diff
@@ -66,11 +66,13 @@ GITLAB_TOKEN ?= ACCTEST1234567890123
 GITLAB_BASE_URL ?= http://127.0.0.1:8080/api/v4
 
 testacc-up: ## Launch a GitLab instance.
```

Comment:
Maybe add a specific target to generate the certificate files and have a proper make dependency on them to not generate them every time?

Reply:
What do you think the benefit might be to not generating them every time? I also tried looking at the openssl flags and couldn't find any options to prevent overwrite if there were pre-existing certs

Reply:
I ended up adding some file tests so hopefully that aligns more to what you were thinking

Reply:
> What do you think the benefit might be to not generating them every time? I also tried looking at the openssl flags and couldn't find any options to prevent overwrite if there were pre-existing certs

It would not hurt, but is just superfluous :) And I had real make goal dependencies in mind ... but again here - not a show stopper ...

Reply:
Somethink li,e this: https://stackoverflow.com/a/21754930

Reply:
Implemented